environment-dev.yml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - pathlib
   - scipy
   - pip
-  - s3fs>=0.4.2
+  - s3fs>=2021.8
```

Comment:
Why do we pin s3fs version only here? We have other env files where we install s3fs.

Reply:
good point, will fix